src/BinSkim.Rules/PERules/BA2025.EnableShadowStack.cs

```diff
@@ -60,6 +60,12 @@ public override AnalysisApplicability CanAnalyzePE(PEBinary target, Sarif.Proper
                 return notApplicable;
             }
 
+            if (portableExecutable.IsARM64Binary)
+            {
+                reasonForNotAnalyzing = MetadataConditions.ImageIsARM64BitBinary;
```

Comment:
>ImageIsARM64BitBinary [](http://example.com/codeflow?start=59&length=21)

Please update casing everywhere to 'Arm', this is the appropriate casing in API contexts. You will see a different casing in some of our file names, etc., but that's ok.